src/ServiceInsight/SequenceDiagram/ModelCreator.cs

```diff
@@ -25,6 +26,7 @@ public ModelCreator(List<StoredMessage> messages, IMessageCommandContainer conta
             this.container = container;
 
             Initialize();
+            OrderHandlers();
```

Comment:
I'm not really keen on the introduction of a second method which reassigns the `handlers` field. Can we encapsulate the ordering inside `Initialize()` and avoid the reassignment?